manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
-    olm.skipRange: ">=4.1.0 <4.1.1"
+    olm.skipRange: ">=4.1.0 <4.1.2"
```

Comment:
@jwforres @adammhaile is ART populating this information automatically now?  or do we still need to be maintaining this?  (also should probably be 4.1.4 not 4.1.2, if we do have to maintain it, since at this point that's what we're starting to build)